docs/conf.py

```diff
@@ -127,85 +129,59 @@
 html_static_path = ["_static"]
 # html_css_files = ["stravalib.css"]
 
-# Short title for the navigation bar.
-html_short_title = "Stravalib Python Package Documentation"
-
 # Instagram always throws 429 so ignore it
 linkcheck_ignore = [r"https://www.instagram.com/accounts/login/"]
-# The name of an image file (relative to this directory) to place at the top
-# of the sidebar.
-# html_logo = None
-
-# The name of an image file (within the static path) to use as favicon of the
-# docs.  This file should be a Windows icon file (.ico) being 16x16 or 32x32
-# pixels large.
-# html_favicon = None
 
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ["_static"]
 
-# If true, "Created using Sphinx" is shown in the HTML footer. Default is True.
-html_show_sphinx = False
-
-# If true, "(C) Copyright ..." is shown in the HTML footer. Default is True.
-html_show_copyright = True
-
-# If true, an OpenSearch description file will be output, and all pages will
-# contain a <link> tag referring to it.  The value of this option must be the
-# base URL from which the finished HTML is served.
-# html_use_opensearch = ''
 
-# This is the file name suffix for HTML files (e.g. ".xhtml").
-# html_file_suffix = None
-
-# Output file base name for HTML help builder.
-htmlhelp_basename = "stravalibdoc"
-
-# -- Options for manual page output ---------------------------------------
+# Example configuration for intersphinx: refer to the Python standard library.
+# intersphinx_mapping = {'http://docs.python.org/': None}
 
-# One entry per manual page. List of tuples
-# (source start file, name, description, authors, manual section).
-man_pages = [
-    ("index", "stravalib", "stravalib Documentation", ["Hans Lellelid"], 1)
-]
+# Default to using the order defined in source.
+autodoc_default_options = {
+    "member-order": "alphabetical",
```

Comment:
sphinx has an exclude_members flag but it seems to only remove generating the stub / text from the members docstring in autosummary. the way to properly remove it is to use the approach below that leverages `auto_skip_members`. there may be a better way but this is the best way i've found so far.